x-pack/plugins/elastic_assistant/server/routes/post_actions_connector_execute.ts

```diff
@@ -5,6 +5,8 @@
  * 2.0.
  */
 
+/* eslint-disable complexity */
```

Comment:
Is this really needed? I took it out and it passed both lint and type check

Reply:
The changes from persistence/streaming might've lowered the count here since it was first committed. Will remove 👍